src/docs/go-grpc/sections/02-try/03-test.mdx

```diff
@@ -0,0 +1,67 @@
+To try the server we will use `grpcurl`, and we will have to pass the flag
+`-cacert` with the root certificate of the `step-ca` instance, if we don't do
+this, the client will not trust the server, and the request will fail:
+
+```bash
+$ grpcurl -cacert ca.crt \
+-d '{"name": "Smallstep"}' myserver.mycompany.net:443 helloworld.Greeter/SayHello
+{
+  "message": "Hello Smallstep"
+} 
+```
+
+If instead of using `grpcurl` we will create a simple gRPC client using Go, we
+also need to load the root certificate to validate the server certificate with
+it:
```

Comment:
```suggestion
Of course, typically you will want to write a gRPC client in Go, instead of using `grpcurl`. 
Like `grpcurl`, the client also needs to trust our root CA certificate to validate the server certificate.
Here's the code for a simple client:
```